### ISSUE TITLE
Please add short troubleshooting section about "Rename button greyed out"

### ISSUE THREAD

**Issue by mohowzeh** [MicrosoftDocs/windows-uwp#3405]
Hi, 

First of all, thanks for this great and useful tool. 

One suggestion is to add a short troubleshooting section explaining why the Rename button remains greyed out in some occasions. 

---
#### Document details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 4c2786bc-ad98-974e-d7b0-a6ce82f8b576
* Version Independent ID: 0be1b400-5349-7dd9-43d2-abd3a91884a6
* Content: [PowerToys PowerRename utility for Windows 10](https://docs.microsoft.com/en-gb/windows/powertoys/powerrename)
* Content Source: [hub/powertoys/powerrename.md](https://github.com/MicrosoftDocs/windows-uwp/blob/docs/hub/powertoys/powerrename.md)
* Product: **dev-environment**
* Technology: **powertoys**
* GitHub Login: @mattwojo
* Microsoft Alias: **mattwoj**

**Comment by mattwojo** [comment]
@dedavis6797 - Known issue ^ ?  Or should this be filed in the PowerToys product repo as a bug?

**Comment by dedavis6797** [comment]
Hi @mohowzeh, can you clarify what button is greyed out? Are you referring to the context menu button when you right click a file?

![image](https://user-images.githubusercontent.com/61330599/137550789-c92b18b6-871f-4d97-8d8c-a9d2d9f9a0df.png)

**Comment by mattwojo** [comment]
Hi @mohowzeh - Closing this for now, please re-open if you can provide more context for this issue.

**Comment by mohowzeh** [comment]
Hi @mattwojo  - sorry for the delayed response. 

The greyed-out button was within the application itself: a list of files was visible, but the "Rename" button was greyed out. 
![PowerRename_greyed-out-button](https://user-images.githubusercontent.com/40325127/139015953-79e85e79-4e49-4509-a740-eae0d3edbdf1.png)

I first couldn't figure out why the button remained greyed out, and I found no documentation on it, hence my suggestion. Eventually I discovered that I had not properly filled in the "Search for:" and "Replace with:" sections. 
Once I had done that, it was OK. 
So the root cause was not the program itself but my own distraction - and I guess there is no easy fix for that ;-)
